readthedocs/embed/v3/views.py

```diff
@@ -214,8 +214,19 @@ def _parse_based_on_doctool(self, page_content, fragment, doctool, doctoolversio
                     # </dl>
 
                     parent_node = node.parent
-                    if 'glossary' in node.parent.attributes.get('class'):
-                        next_node = node.next
+                    if "glossary" in node.parent.attributes.get("class"):
+                        # iterate through child and next nodes
+                        traverse = node.traverse()
+                        iteration = 0
+                        while iteration < 5:
+                            next_node = next(traverse, None)
+                            # TODO: Do we need to support terms with missing descriptions?
+                            # This will not produce correct results in this case.
+
+                            # Stop at the next 'dd' node, which is the description
+                            if iteration >= 5 or (next_node and next_node.tag == "dd"):
+                                break
```

Comment:
Isn't the `iteration >= 5` already provided by the `while iteration < 5`?

Reply:
indeed, the first part of the condition can be removed, but it's also harmless.